docs/migration-guide.md

```diff
@@ -24,7 +24,7 @@ for users to migrate effectively.
 
 * [Spark Core](core-migration-guide.html)
 * [SQL, Datasets, and DataFrame](sql-migration-guide.html)
-* [Structured Streaming](ss-migration-guide.html)
+* [Structured Streaming](/streaming/ss-migration-guide.html)
```

Comment:
We might need to add canonical links from the old URLs to the updated ones.

Reply:
I actually think that a plain redirect (which I can do via a `redirect:`) tag in the front-matter, as per Kent's suggestion, would be fine here: https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls#comparison-of-canonicalization-methods

It says:

> Use redirects to tell Googlebot that a redirected URL is a better version than a given URL. Use this only when deprecating a duplicate page.

Which I think is our use-case here, since this page is being deprecated.